web/src/app/(main)/governor/[governorAddress]/ActiveLists/ExamineList.tsx

```diff
@@ -0,0 +1,88 @@
+import { useState } from "react";
+
+import { Card, CustomAccordion, DraggableList } from "@kleros/ui-components-library";
+import clsx from "clsx";
+
+import { AddressOrName, IdenticonOrAvatar } from "@/components/ConnectWallet/AccountDisplay";
+import DisplayCard from "@/components/ListDisplayCard";
+import Status from "@/components/Status";
+
+import Calendar from "@/assets/svgs/icons/calendar.svg";
+
+import { List } from "@/consts/mockLists";
+
+export const AccordionBody: React.FC<{ transactions: List["transactions"] }> = ({ transactions }) => {
+  const [selectedTxn, setSelectedTxn] = useState<List["transactions"][number]>(transactions[0]);
+
+  return (
+    <div
+      className={clsx(
+        "w-full grid grid-cols-1 md:grid-cols-[minmax(300px,_554px)_minmax(300px,_465px)]",
+        "place-content-center items-stretch",
+        "pt-2 md:p-2 lg:p-6 gap-2.5 md:gap-6"
+      )}
+    >
+      <Card className="size-full py-4 flex flex-col">
+        <DraggableList
+          dragDisabled
+          deletionDisabled
+          defaultSelectedKeys={[0]}
+          disallowEmptySelection
+          className="border-none flex-1 size-full bg-klerosUIComponentsWhiteBackground"
+          items={transactions.map((txn) => ({ name: txn.name, id: txn.index, value: txn }))}
+          selectionCallback={(selected) => setSelectedTxn(selected.value)}
+        />
+      </Card>
+      <div className="flex flex-col gap-4 md:gap-4">
+        <DisplayCard label="Contract Address" value={selectedTxn.to} />
+        <DisplayCard label="Value" value={selectedTxn.value.toString()} />
+        <DisplayCard label="Data Input" value={selectedTxn.data} />
+        <DisplayCard label="Decoded Input" value={selectedTxn.decodedInput} />
+      </div>
+    </div>
+  );
+};
+
+interface IAccordionTitle extends Pick<List, "submitter" | "createdOn" | "status"> {
+  numberOfTxns: number;
+}
+
+export const AccordionTitle: React.FC<IAccordionTitle> = ({ submitter, createdOn, numberOfTxns, status }) => {
+  return (
+    <div className="flex flex-wrap gap-2 md:gap-8 items-center">
+      <h3 className="text-base text-klerosUIComponentsPrimaryText font-semibold">List</h3>
+      <span className="text-sm text-klerosUIComponentsPrimaryPurple">{numberOfTxns} Txns</span>
+      <small className="flex gap-2 items-center text-xs text-klerosUIComponentsSecondaryText max-md:basis-full">
+        <Calendar className="size-3.5" />
+        {createdOn}
+      </small>
+      <Status {...{ status }} />
+      <div className="flex gap-2 items-center max-md:basis-full">
+        <IdenticonOrAvatar size="16" address={submitter} />
+        <AddressOrName className="text-sm text-klerosUIComponentsPrimaryText" address={submitter} />
+      </div>
+    </div>
+  );
+};
```

Comment:
if these two components are not going to be imported anywhere else, I would remove the `export` I think I remember reading somewhere that multiple component exports on the same file can trip the dev server's fast refresh 🤔